src/Header/ResponseEncoder.h

```diff
@@ -5,9 +5,10 @@
 #ifndef GROUP4_SRC_RESPONSEENCODER_H_
```

Comment:
should update header in header branch

Reply:
It is usual to put header and implementation in the same commit when the changes are related.